actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -795,7 +795,7 @@ def init_with(coder) # :nodoc:
         @permitted  = coder.map["ivars"][:@permitted]
       when "!ruby/object:ActionController::Parameters"
         # YAML's Object format. Only needed because of the format
-        # backwardscompability above, otherwise equivalent to YAML's initialization.
+        # backwards compatibility above, otherwise equivalent to YAML's initialization.
```

Comment:
I’d rather keep the git history here.

Reply:
I think it is fine here. That word doesn't exist 😄